index.html

```diff
@@ -268,8 +269,16 @@ <h2 class="work-title">My Recent Works</h2>
     <!-- about section -->
     <section class="about-section">
       <div class="about-text">
-        <img src="icons/desktopimage/illustration-about me-1-desktop@2x.png" alt="" class="image1">
-        <img src="icons/desktopimage/illustration-about-me-2-desktop@2x.png" alt="" class="image2">
+        <img
+          src="icons/desktopimage/illustration-about me-1-desktop@2x.png"
+          alt=""
+          class="image1"
+        />
+        <img
+          src="icons/desktopimage/illustration-about-me-2-desktop@2x.png"
+          alt=""
+          class="image2"
+        />
 
         <h1 class="primary-text">About me</h1>
```

Comment:
![Screenshot from 2022-10-04 21-22-17](https://user-images.githubusercontent.com/93039359/193874628-c520191a-cb5e-43e0-8487-d267f8cde85a.png)

- [ ] Your image is covering a card in the mobile version which is not good for accessibility. Kindly arange this image in such a way to show your card properly :blush: